### DISCUSSION TITLE
How to install a specific version of bun on Windows?

### DISCUSSION THREAD

**Discussion by HuakunShen** [oven-sh/bun#14889] [Q&A] [answered]
I had problem with latest 1.1.33 on Windows, and have to downgrade to 1.1.27.
Is there a one liner to do that?
There is instruction for bash, but not Windows. https://bun.sh/docs/installation

**Comment by alexeyten** [comment, marked as answer]
https://bun.sh/docs/installation#installing-a-specific-version-of-bun-on-windows ?

```
iex "& {$(irm https://bun.sh/install.ps1)} -Version 1.1.27"
```

**Reply by HuakunShen** [reply]
Thanks!
